### PR TITLE
Add separate lobbyist download page

### LINKED ISSUE #221: Move lobbyist data to separate download page

### PR DESCRIPTION
**Pull request by antidipyramid**
## Overview

This PR adds a separate lobbyist download page. Since there are two download pages now (one for campaign finance and another for lobbyists) I renamed the templates and views accordingly.

Closes #221 

### Demo

Screenshot 2024-10-23 at 10 24 59 AM

### Notes

Text content can be added to the page in the admin.

## Testing Instructions

* Verify that the new download page works

### CODE REVIEW

**Comment by fgregg** [comment]
@antidipyramid why isn't the heroku preview working?

File: pages/templates/lobbyist_downloads.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        </div>
+
+      </div>
+    </div>
+
+  </div>
+{% endblock %}
+
+{% block extra_js %}
+  <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
```
Comment (fgregg) [inline pages/templates/lobbyist_downloads.html:60]: why do we need these date pickers on this page?

File: camp_fin/urls.py [hunk 2]
```diff
@@ -107,7 +109,14 @@
...
     ),
-    path("downloads/", DownloadView.as_view(), name="downloads"),
+    path(
+        "camp-fin-downloads/",
+        CampaignFinanceDownloadView.as_view(),
+        name="camp-fin-downloads",
+    ),
+    path(
+        "lobbyist-downloads/", LobbyistDownloadView.as_view(), name="lobbyist-downloads"
+    ),
```
Comment (fgregg) [inline camp_fin/urls.py:119]: let's add a 301 redirect from downlods to camp-find-downloads.

**Comment by antidipyramid** [comment]
@fgregg [Looks like the app hasn't finished deploying.](https://dashboard.heroku.com/apps/openness-pro-feature-37-atzawt)

**Review by fgregg — CHANGES_REQUESTED** [review]

File: pages/templates/lobbyist_downloads.html (hunk 1 above)
Comment (antidipyramid) [reply pages/templates/lobbyist_downloads.html]: We don't need them-- thanks for flagging.

File: camp_fin/urls.py [hunk 3]
```diff
@@ -104,6 +105,7 @@
         name="expenditure-detail",
     ),
     path("committees/", CommitteeList.as_view(), name="committee-list"),
+    path("downloads/", downloads_redirect_view, name="downloads"),
```
Comment (fgregg) [inline camp_fin/urls.py:108]: i think you can just do

```python
path("downloads/",
     HttpResponsePermanentRedirect(reverse("camp-fin-downloads")),
     name="downloads"),
```

seems clearer than creating a custom view that's just wrapping that up.

**Review by fgregg — CHANGES_REQUESTED** [review]

File: camp_fin/urls.py (hunk 3 above)
Comment (antidipyramid) [reply camp_fin/urls.py]: Agreed. Doing it this way threw an error, so I used a [RedirectView](https://docs.djangoproject.com/en/5.1/ref/class-based-views/base/#redirectview).

**Review by fgregg — APPROVED** [review]

**Pull request merged**